utils/config.py

```diff
@@ -36,11 +33,22 @@ def network_name() -> Optional[str]:
 elif network_name() in ("sepolia", "sepolia-fork"):
     print(f'Using {color("yellow")}config_sepolia.py{color} addresses')
     from configs.config_sepolia import *
+elif network_name() in ("hoodi", "hoodi-fork"):
+    print(f'Using {color("cyan")}config_hoodi.py{color} addresses')
+    from configs.config_hoodi import *
 else:
     print(f'Using {color("magenta")}config_mainnet.py{color} addresses')
     from configs.config_mainnet import *
 
 
+def get_vote_duration() -> int:
+    """
+    Get the vote duration in seconds.
+    """
+    voting = interface.Voting(VOTING)
+    return voting.voteTime() + voting.objectionPhaseTime()
```

Comment:
Just `voting.voteTime()` is enough. Objection phase already included in the voteTime

See here for details:
https://github.com/lidofinance/aragon-apps/blob/master/apps/voting/contracts/Voting.sol#L769-L780